indico/modules/rb_new/controllers/backend/rooms.py

```diff
@@ -76,9 +79,12 @@ def _process(self, args):
                 raise UnprocessableEntity('Required data to filter by availability is not present')
         else:
             availability = not only_unavailable
-
         search_query = search_for_rooms(args, allow_admin=admin_override_enabled, availability=availability)
-        room_ids = [id_ for id_, in search_query.with_entities(Room.id)]
+        rooms = [(id_, room_name) for id_, room_name, in search_query.with_entities(Room.id, Room.full_name)]
```

Comment:
what about using a dict here? That would let you simplify some of the code below

Reply:
how would it be simpler then?

Reply:
`room_ids = rooms.keys()` for example

Reply:
I see it now, makes sense, I'll change it